string label
int32 class_id
int32 x_center
int32 y_center
int32 w
int32 h
int32 size
float32 conf
